handlers/templates/pages/guest-link-index.html

```diff
@@ -82,6 +86,50 @@
           });
       });
     });
+
+    document
+      .querySelectorAll('[aria-label="Enable"], [aria-label="Disable"]')
+      .forEach((toggleBtn) => {
+        toggleBtn.addEventListener("click", () => {
+          const id = toggleBtn.getAttribute("pico-link-id");
+          // switch enable/disable
+          if (toggleBtn.getAttribute("aria-label") === "Enable") {
+            guestLinkEnable(id)
+              .then(() => {
+                const button = toggleBtn;
+                button.classList.remove("is-danger");
+                button.classList.add("is-success");
+                button.setAttribute("aria-label", "Disable");
+                button.innerHTML =
```

Comment:
I try to avoid assigning `.innerHTML` since it's typically more powerful than we need. In this case, it looks like the only change is adding or removing `fa-check-square`. Can we do that with `classList` instead?